resources/energyplus/ProposedEnergy+.idd

```diff
@@ -1,4 +1,4 @@
-!IDD_Version 22.2.0
+!IDD_Version 23.1.0
```

Comment:
Side Note, but I'd find it clearer if you added all new objects to ProposedEnergy+.idd when you do this initial PR. The feature ones can build up upon it.

Reply:
Should with add this item to the https://github.com/NREL/OpenStudio/wiki/EnergyPlus-Version-Change wiki?

Reply:
Good idea, I added a clarification in the first bullet point that did mention "copying over changes".